packages/grafana-flamegraph/src/FlameGraphContainer.tsx

```diff
@@ -234,25 +255,55 @@ function useColorScheme(dataContainer: FlameGraphDataContainer | undefined) {
   return [colorScheme, setColorScheme] as const;
 }
 
-function getStyles(theme: GrafanaTheme2, vertical?: boolean) {
+function getStyles(theme: GrafanaTheme2) {
   return {
     container: css({
       label: 'container',
+      overflow: 'auto',
       height: '100%',
-      display: vertical ? 'block' : 'flex',
+      display: 'flex',
       flex: '1 1 0',
       flexDirection: 'column',
       minHeight: 0,
       gap: theme.spacing(1),
     }),
     body: css({
       label: 'body',
-      display: 'flex',
       flexGrow: 1,
+    }),
+
+    tableContainer: css({
+      // This is not ideal for dashboard panel where it creates a double scroll. In a panel it should be 100% but then
+      // in explore we need a specific height.
+      height: 800,
+    }),
+
+    horizontalContainer: css({
+      label: 'horizontalContainer',
+      display: 'flex',
       minHeight: 0,
-      height: vertical ? undefined : '100vh',
-      flexDirection: vertical ? 'column-reverse' : 'row',
+      flexDirection: 'row',
       columnGap: theme.spacing(1),
+      width: '100%',
+    }),
+
+    horizontalGraphContainer: css({
+      flexBasis: '50%',
+    }),
+
+    horizontalTableContainer: css({
+      flexBasis: '50%',
+      maxHeight: 800,
+    }),
+
+    verticalContainer: css({}),
+
+    verticalGraphContainer: css({
+      marginBottom: theme.spacing(1),
+    }),
+
+    verticalTableContainer: css({
+      height: 800,
```

Comment:
Note this was set to 600 previously so not sure what the significance of the extra 200px is?

Reply:
This was the intended previous maxHeight in code that in the end wasn't working properly so just moved it here. Otherwise totally arbitrary tbh.